docs/user-guide.md

```diff
@@ -200,19 +201,30 @@ When the debugger is attached, the Visual Studio Code [debug options](https://co
 
 ![Debug Mode](/docs/user-guide-screenshots/debugger_mode.png)
 
-## Manually add your Liberty project to the dashboard
+## Manually add your Liberty project to the Liberty dashboard
 
-In the event that your Liberty project is not automatically detected by the Liberty dashboard, you can manually add your Liberty project to the Liberty dashboard. To manually add your Liberty project to the Liberty dashboard, right-click into an empty space in the project explorer and select **Add project to Liberty Dashboard** or select the **Liberty: Add project to the tool window** command in the command palette, followed by your application.
+In the event that your Liberty project is not automatically detected by the Liberty dashboard, you can manually add your Liberty project to the Liberty dashboard in one of four ways:<br>
+1. Right-click into an empty space in the Explorer view and select **Add project to Liberty Dashboard**.<br>
+  ![Add Liberty project from Explorer view empty space](/docs/user-guide-screenshots/add_from_empty_space.png)<br>
+2. If you have a workspace open, you can right-click into an empty space in the Explorer view as shown above, or you can right-click on a project root folder in the Explorer view and select **Add project to Liberty Dashboard**.<br>
+  ![Add Liberty project from Explorer view project root](/docs/user-guide-screenshots/add_from_project_root_folder.png)<br>
+3. Click on the **'+'** icon in the Liberty dashboard.<br>
+  ![Add Liberty project from Liberty dashboard](/docs/user-guide-screenshots/add_from_liberty_dashboard.png)<br>
+4. Use the Command Palette to select the **Liberty: Add project to Liberty Dashboard** command.<br>
+  ![Add Liberty project from Command Palette](/docs/user-guide-screenshots/add_from_command_palette.png)<br>
 
-![Liberty add project to tool window](/docs/user-guide-screenshots/add_liberty_project.png)
+In all the cases above, you will be prompted in the Command Palette to select a project to add from a list of projects that are currently not displayed in the Liberty dashboard.<br>
+![Liberty add project selection](/docs/user-guide-screenshots/add_project_selection.png)<br>
 
-You are prompted with a list of projects that are not already displayed in the in the Liberty dashboard.
+### Manually remove project
 
-![Liberty add project to tool window selection](/docs/user-guide-screenshots/add_liberty_project_2.png)
+To remove manually added Liberty projects from the Liberty dashboard, you can follow the same steps shown above for adding a project with the following differences:
```

Comment:
It took me a few minutes to realize why the first two items are the same. Maybe this sentence should be reworded in part "you can follow one of the same four steps shown above."

Reply:
Thanks @turkeylurkey for the suggestion. This should be taken care of with @dmuelle's edits.